lib/utils/collect-keys.js

```diff
@@ -37,96 +48,123 @@ function getKeyFromVAttribute (node) {
+    tokens: true,
+    comment: true,
+    eslintVisitorKeys: true,
+    eslintScopeManager: true,
+    filePath: effectiveFilename
+  })
+  try {
+    const parseResult = (typeof parser.parseForESLint === 'function')
+      ? parser.parseForESLint(text, parserOptions)
+      : { ast: parser.parse(text, parserOptions) }
+    return collectKeysFromAST(parseResult.ast, parseResult.visitorKeys)
+  } catch (_e) {
+    return []
+  }
 }
 
 /**
- * Collect the used keys.
+ * Collect the used keys from files.
  * @returns {string[]}
  */
-function collectKeys (files, extensions) {
-  debug('collectKeys', files, extensions)
-
-  // setup linter options
-  const config = {
-    parser: 'vue-eslint-parser',
-    parserOptions: {
-      ecmaVersion: 2018,
-      sourceType: 'module',
-      ecmaFeatures: { jsx: true }
-    },
-    extensions,
-    env: { browser: true, es6: true },
-    rules: { [INTERNAL_RULE_KEY]: 'error' }
-  }
+function collectKeysFromFiles (files, extensions) {
+  debug('collectKeysFromFiles', files, extensions)
 
-  // setup linter
-  const linter = new Linter()
-  linter.defineParser('vue-eslint-parser', require(require.resolve(config.parser)))
-  linter.defineRule(INTERNAL_RULE_KEY, { create })
+  const cliEngine = new CLIEngine()
 
   const results = new Set()
 
   // detect used lodalization keys with linter
-  for (const { filename, ignored } of listFilesToProcess(files, config)) {
+  for (const { filename, ignored } of listFilesToProcess(files, { extensions })) {
     debug(`Processing file ... ${filename}`)
 
     if (ignored) { continue }
+    if (!extensions.includes(extname(filename))) { continue }
 
     const text = readFileSync(resolve(filename), 'utf8')
 
-    for (const usedKey of processText(text, config, filename, linter)) {
+    for (const usedKey of collectKeysFromText(text, filename, cliEngine)) {
       results.add(usedKey)
     }
   }
 
   return [...results]
 }
 
-module.exports = collectKeys
+/**
+ * Collect the used keys from Program node.
+ * @returns {string[]}
+ */
+function collectKeysFromAST (node, visitorKeys) {
+  debug('collectKeysFromAST')
+
+  const results = new Set()
+  /**
+   * @param {Node} node
+   */
+  function enterNode (node) {
+    if (node.type === 'VAttribute') {
+      if (node.directive && (node.key.name.name === 't' || node.key.name === 't' /* parser v5 */)) {
+        debug("call VAttribute[directive=true][key.name.name='t'] handling ...")
+        const key = getKeyFromVDirective(node)
+        if (key) {
+          results.add(key)
+        }
+      }
+    } else if (node.type === 'CallExpression') {
+      debug('CallExpression handling ...')
+      const key = getKeyFromCallExpression(node)
+      if (key) {
+        results.add(key)
+      }
+    }
+  }
+
+  if (node.templateBody) {
+    vueESLintParser.AST.traverseNodes(node.templateBody, {
+      enterNode,
+      leaveNode () {}
+    })
+  }
+  vueESLintParser.AST.traverseNodes(node, {
+    visitorKeys: visitorKeys,
```

Comment:
You can use `shorthand prop name` sytanx.
```js
vueESLintParser.AST.traverseNodes(node, {
    visitorKeys
```

Reply:
I fixed this. I also enabled the "object-shorthand" rule in `.eslintrc`.